conbench/templates/index.html

```diff
@@ -25,16 +25,16 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
 <div class="row">
     <div class="col-md-12">
         <table id="runs" class="table table-striped table-bordered table-hover">
-          <caption>Runs</div>
+          <caption>Runs
```

Comment:
Oh. How did that even work/render before? Reminds of many years ago when I was manually writing quite a bit of HTML, always involving https://validator.w3.org/ before being confident about validity.

Reply:
Code editors at rescue :D